cgi/product_multilingual.pl

```diff
@@ -670,16 +670,17 @@ ($$)
 		${field}.dropdown.show.call(${field}, value); // render the suggestions dropdown
 	})
 });
-${field}.on('add', function(tag) {
-	let obj = JSON.parse(window.localStorage.getItem("po_last_tags"));
+${field}.on('add', function(e) {
+	const obj = JSON.parse(window.localStorage.getItem("po_last_tags"));
 
-	if (obj == null) {
+	const tag = e.detail.data.value;
+	if (obj === null) {
 		obj = {"${field}": [tag]}
-	} else if (obj["${field}"] == null) {
+	} else if (obj["${field}"] === null) {
```

Comment:
Here i would also like to check if it's = to "" or to undefined.
>There are only six falsy values in JavaScript you should be aware of:
    false — boolean false
    0 — number zero
    “” — empty string
    null
    undefined
    NaN — Not A Number